src/web-ui/src/analytics/AnalyticsHandler.js

```diff
@@ -197,6 +197,7 @@ export const AnalyticsHandler = {
                 category: product.category,
                 image: product.image,
                 feature: feature,
+                recipe: recipe,
```

Comment:
Please add 'recipe' to attributes for AmplifyAnalytics.record call above too for consistency.